.github/workflows/sync-fork.yml

```diff
@@ -0,0 +1,17 @@
+name: Sync Fork
+
+on:
+  schedule:
+    - cron: '0 0 * * 0' # every Sunday
+
+jobs:
+  sync:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: tgymnich/fork-sync@v1.6.3
```

Comment:
is it "safe" to use other people's actions?

Reply:
Shopify already uses 3rd party Github Actions in other repos